source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/MessageHub/Models/AvailableOperationReceiptData/AvailableOperationRejectionsFactoryTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoFixture.Xunit2;
+using Energinet.DataHub.Core.TestCommon.AutoFixture.Attributes;
+using FluentAssertions;
+using GreenEnergyHub.Charges.Application.Charges.Events;
+using GreenEnergyHub.Charges.Application.Messaging;
+using GreenEnergyHub.Charges.Domain.Dtos.SharedDtos;
+using GreenEnergyHub.Charges.Domain.MarketParticipants;
+using GreenEnergyHub.Charges.Infrastructure.Core.Cim.MarketDocument;
+using GreenEnergyHub.Charges.MessageHub.Models.AvailableOperationReceiptData;
+using GreenEnergyHub.Charges.Tests.MessageHub.Models.Shared;
+using Moq;
+using NodaTime;
+using Xunit;
+
+namespace GreenEnergyHub.Charges.Tests.MessageHub.Models.AvailableOperationReceiptData
+{
+    public class AvailableOperationRejectionsFactoryTests
```

Comment:
```suggestion
    public class AvailableChargePriceOperationRejectionsFactoryTests
```
Remember filename, too.